internal/database/actions.go

```diff
@@ -276,7 +275,7 @@ func CreateResult(sess db.Session, homePath string, testGroupName string, networ
 }
 
 // CreateNetwork will create a new network in the network table
-func CreateNetwork(sess db.Session, loc enginex.LocationProvider) (*Network, error) {
+func CreateNetwork(sess db.Session, loc engine.LocationProvider) (*Network, error) {
```

Comment:
Can you update the codebase to use `engine.LocationProvider` and delete `enginex.LocationProvider`?